### DISCUSSION TITLE
I need Github student bag for help with Twilio

### DISCUSSION THREAD

**Discussion by 1746705990** [community/community#55329] [GitHub Education]
### Hi everyone,

It is worth affirming that my Github student bag has been applied for, which is very happy.

But there is one project in Github's student bag that I cannot use, which is Twilio's rights. To be precise, when I registered for a Twilio account and prepared to try out the Twilio benefits Github brought me, my Twilio account was banned for no reason. When I try to submit a work order and apply for Twilio to lift the ban, they always refuse without any reason. This situation has been going on for a week, which makes me very uncomfortable.

Because I was preparing to learn about using Python code to call interfaces to send and receive messages, their blockade of my account directly rejected my learning plan, and I had to postpone the plan, which made me very angry.

Because this right belongs to you, Github brought it to me, but Twilio can intervene at will. I have to believe that this is a small trick made by Twilio to evade responsibility, which makes me very angry, especially when I have been communicating for a long time without any news. Since this belongs to the Github student package and I am still a college student, I have the right to use Twilio.

That's all. I hope you can help me, as my learning requires the communication interface of Twilio. Thank you for your attention.

I will attach the email sent to me by Twilio！ @twilio-dx @Twilio
![image](https://github.com/community/community/assets/126673150/3754195c-6f5f-49be-a307-f924ab530bc5)
![image](https://github.com/community/community/assets/126673150/3523ece5-23dd-471f-8ca5-11545c545022)
I have already indicated in my email with him that I have obtained the Github student package, but he still does not grant me the suspension of my account.I told him that I am a student and an individual developer, but I can't help it.I think he just doesn't want me to try Github student bags!!!😠😠😠

_No response_

**Comment by 1746705990** [comment]
@joenash
Sorry to bother you, can you help me?

**Reply by joenash** [reply]
Sorry, I no longer work at Twilio. Best of luck!

**Reply by 1746705990** [reply]
Okay, but could you give me some advice?
Because my study plan requires the Twilio interface, but now my account cannot even be successfully registered.
 Once my account was registered, it was marked with fraudulent information and then locked.😭😭😭😭😭

**Comment by ryanlua** [comment]
Hi @1746705990,

It's quite an unfortunate situation you are in. You can [request GitHub support](https://support.github.com/request) and hope they try to assist you but I highly doubt they will be able to help you as they do not own or moderate in correlation to Twilio.

I don't think they will be any help from GitHub but you can try to reach out to support agent at Twilio.

**Reply by 1746705990** [reply]
Thank you for your discussion. 😄
So far, I have submitted a work order to Github support, but I have not received a reply yet. 😫
As for Twilio, I sent another email asking for the suspension of my account, which also went unanswered. I can only wait anxiously!😡

**Comment by samus-aran** [comment]
Hi @1746705990 

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

Here is the similar post you created: https://github.com/orgs/community/discussions/55740

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
